Jenkinsfile

```diff
@@ -122,6 +125,10 @@ def runIntegrationTest(String description, String kubeprodArgs, String ginkgoArg
                         junit 'junit/*.xml'
                     }
                 }
+
+                withEnv(["PATH+KUBECFG=${tool 'kubecfg'}"]) {
+                    sh "kubecfg -v delete kubeprod-manifest.jsonnet"
```

Comment:
Is this useful, or does it just introduce delays?  We're about to delete the entire cluster anyway, it would be nice if that was sufficient..

(Or: if we _need_ to delete all the contents of the cluster first, then presumably we need to wait for the implications of that to complete - wait until PVCs are unbound and PVs are removed, loadbalancers are deleted, etc.  Just doing this delete here and then immediately continuing seems inadequate.)

Reply:
Deleting the cluster using `eksctl` doesn't seem enough: it does not remove any ELB interfaces which prevent the CloudFormation stacks from being deleted. If we are to wait for the cluster to be deleted, how can I do that?

Reply:
Your `waitForDestroy()` looks like a reasonable solution.